### PR TITLE
Chronos: add chronos known issue

### PR DESCRIPTION
**Pull request by Rebekk**

### CODE REVIEW

**Comment by TheaperDeng** [comment]
please put a preview readthedoc page in comments

**Comment by Rebekk** [comment]
Here's the preview readthedoc page: https://chronos-issue.readthedocs.io/en/add-doc/doc/Chronos/Overview/chronos_known_issue.html

File: docs/readthedocs/source/doc/Chronos/Overview/chronos_known_issue.md [hunk 1]
```diff
@@ -0,0 +1,88 @@
+# Chronos Known Issue
+## **1. Issue 1**
+**Problem description**
+
+Numpy.ndarray size changed, may indicate binary incompatibility. Expected 88 from C header, got 80 from PyObject. 
+
+**Version**
```
Comment (TheaperDeng) [inline docs/readthedocs/source/doc/Chronos/Overview/chronos_known_issue.md:7]: we don't need this version section

File: docs/readthedocs/source/doc/Chronos/Overview/chronos_known_issue.md [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+* `conda install –c conda-forge pycocotools`
+
+---------------------------
+
+## **2. Issue 2**
+**Problem description**
+
+NotImplementedError: Cannot convert a symbolic Tensor (encoder_lstm_8/strided_slice:0) to a numpy array. 
+
+**Version**
```
Comment (TheaperDeng) [inline docs/readthedocs/source/doc/Chronos/Overview/chronos_known_issue.md:25]: we don't need this version section

File: docs/readthedocs/source/doc/Chronos/Overview/chronos_known_issue.md [hunk 3]
```diff
@@ -0,0 +1,88 @@
...
+
+---------------------------
+
+## **3. Issue 3**
+
+**Problem description**
+
+StanModel object has no attribute 'fit_class', cause of pip, may be. 
+
+**Version**
```
Comment (TheaperDeng) [inline docs/readthedocs/source/doc/Chronos/Overview/chronos_known_issue.md:41]: we don't need this version section

File: docs/readthedocs/source/doc/Chronos/Overview/chronos_known_issue.md [hunk 4]
```diff
@@ -0,0 +1,88 @@
...
+ Sed: error while loading shared libraries: libunwind.so.8: cannot open shared object file: No such file or directory.
+> Only happens when you run `source bigdl-nano-init`. 
+
+**Solution**
+
+* `apt-get install libunwind8-dev` 
+
+---------------------------
+
+## **6. Issue 6**
```
Comment (TheaperDeng) [inline docs/readthedocs/source/doc/Chronos/Overview/chronos_known_issue.md:79]: remove this issue

**Pull request merged**